shared/clearwater/sticky_component.rb

```diff
@@ -1,79 +1,48 @@
 require 'clearwater/component'
-require 'clearwater/black_box_node'
 
 module Clearwater
   class StickyComponent
     include Clearwater::Component
 
-    attr_accessor :props, :renderer
-    attr_reader :children
-
-    def self.render props=nil, children=nil
-      unless `props === nil || props.$$is_hash`
-        children = props
-        props = nil
+    class << self
+      alias_method :_new, :new
+      def new(*args)
```

Comment:
One of the reasons I used `.render` instead of `.new` was so you could choose use a `StickyComponent` instance as a regular component. I don't know if that's a reasonable thing to do, though. What are your thoughts on that?